config/locales/client.en.yml

```diff
@@ -3862,7 +3862,7 @@ en:
         edit_action: "Edit"
         edit_anonymous: "Sorry, but you need to be logged in to edit this post."
         flag_action: "Flag"
-        flag: "privately flag this post for attention or send a private notification about it"
```

Comment:
Same thing here - I think we want to keep "privately" or find way to make clear that flags are not public, and are only seen by moderators and admins.